src/Schema/Table.php

```diff
@@ -109,7 +109,7 @@ protected function _getMaxIdentifierLength()
             return $this->_schemaConfig->getMaxIdentifierLength();
         }
 
-        return 63;
+        return 30;
```

Comment:
use original/63 length when merging into v4.0.x

related with https://github.com/doctrine/dbal/pull/5493

Reply:
@morozov is it worth to pursue this change for v3.4 - <4.0 because of Oracle DB v11? Or should I fix the tests for Oracle to pass instead only?

Reply:
Which test does this issue affect? Tweaking a test to make it pass despite the bug is wrong. You mark it incomplete instead.

Reply:
see https://github.com/doctrine/dbal/runs/7508984916

Reply:
fixed in https://github.com/doctrine/dbal/pull/5490